systemtest/src/main/java/io/strimzi/systemtest/resources/jaeger/SetupJaeger.java

```diff
@@ -142,14 +142,16 @@ private static void deployJaegerContent() {
     private static void deployJaegerOperator() {
         LOGGER.info("=== Applying Jaeger Operator install files ===");
 
+        // create namespace `cert-manager` and add it to stack, to collect logs from it
```

Comment:
It's not Namespace for cert-manager, or?

Reply:
typo, fixed